charts/consul/templates/dns-proxy-deployment.yaml

```diff
@@ -0,0 +1,198 @@
+{{- if (or (and (ne (.Values.dns.proxy.enabled | toString) "-") .Values.dns.proxy.enabled) (and (eq (.Values.dns.proxy.enabled | toString) "-") .Values.global.enabled)) }}
+{{- if not .Values.connectInject.enabled }}{{ fail "connectInject.enabled must be true" }}{{ end -}}
+{{ template "consul.validateRequiredCloudSecretsExist" . }}
+{{ template "consul.validateCloudSecretKeys" . }}
```

Comment:
```suggestion
```

I think these were remnants that got included in the RFC and POC. We don't want to require mesh.  We also don't want to add more cloud stuff.